cwizard2011/maintenace-tracker-app
# #157721401 setup postgresql database

#### What does this PR do?
Setup the app to communicate with PostgreSQL

#### Description of Task to be completed?
The app should send data and retrieve data from the database

#### How should this be manually tested?
- clone this app using ```git clone https://github.com/cwizard2011/maintenace-tracker-app.git```
- navigate to the app directory using ```cd maintenace-tracker-app``` in your terminal
- run ```npm install```
- setup your database url following the guide in ```.env.example```
- In your database run query ```CREATE TYPE role AS ENUM ('users', 'admin')``` and ```CREATE TYPE status AS ENUM ('new', 'approved', 'pending', 'rejected', 'resolved') ``` to create your ENUMERATION data type
- run ```node server/models/database.js``` to create your user table and request table in the database
- check your database to verify the created table
#### Any background context you want to provide?
The app is setup using 
-node version 9.11.1,
- pg version 6.1.0
- elephantsql

#### What are the relevant pivotal tracker stories?
#157721401
